libraries/utils.py

```diff
@@ -1,5 +1,7 @@
 import random
 import string
+import re
+
```

Comment:
is that a typo? I don't see it being used

Reply:
it is, line 195.